### ISSUE TITLE
'samlSignedRequestEnabled' argument is missing from okta_app_saml resource

### ISSUE THREAD

**Issue by eduhamel** [okta/terraform-provider-okta#1401]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The new 'samlSignedRequestEnabled' argument, that can be configured by API, is not yet implemented for the okta_app_saml resource.
### New or Affected Resource(s)

- okta_app_saml

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

- #0000

**Comment by monde** [comment]
@eduhamel are you attempting to configure an OIN app with the `okta_app_saml` resource? Also, are you inspecting the private API calls that are made in the Admin UI and inferring that this information should be in the Okta TF provider? Just looking for clarification as the private API the Admin UI uses is not the same as the public management API.

**Comment by eduhamel** [comment]
@monde this attribute can be configured for all saml apps, it was added recently in Okta. Before, it was not possible to sign SAML requests. Now it can be configured by activating it (as for SLO, it requires the SP certificate to be uploaded as well)

**Comment by monde** [comment]
got it, thanks @eduhamel . I'm not seeing `samlSignedRequestEnabled` in the public API documentation yet https://developer.okta.com/docs/reference/api/apps/

**Comment by monde** [comment]
Okta internal reference: https://oktainc.atlassian.net/browse/OKTA-559581

**Comment by eduhamel** [comment]
hey @monde, do you have any idea when this will be implemented? we're currently moving forward with applications migration and this is a blocking point. For SAML apps, we would like to upload the SP certificate along with activating the signing of SAML requests. For now, if we want to upload the SP certificate, we have to configure SLO (which we do not want to) and then manually select the signing SAML requests option, but even by doing so, it is deleted at the next Terraform apply (the activation of SAML requests signing)

**Comment by monde** [comment]
@eduhamel can you open a support ticket asking for this issue to be given priority? That will help pull the levers that decide what work is given priority.  https://support.okta.com/

**Comment by monde** [comment]
Discussed this in our weekly triage meeting today.

**Comment by monde** [comment]
Released in [v3.43.0](https://github.com/okta/terraform-provider-okta/releases/tag/v3.43.0)
